src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -576,6 +576,11 @@ void QMCFixedSampleLinearOptimizeBatched::process(xmlNodePtr q)
   m_param.add(OutputMatricesHDF, "output_matrices_hdf", {"no", "yes"});
   m_param.add(FreezeParameters, "freeze_parameters", {"no", "yes"});
 
+  m_param.add(ev_solver_, "ev_solver",
```

Comment:
Would like to call it `eigensolver` than `ev_solver`.

Reply:
Would you like to add the manual entry in this PR or later? If later, I would like to approve the current PR.

Reply:
Either way is fine. If you want to keep this moving, I can add the manual entry later.